test/test_facebook.py

```diff
@@ -197,5 +199,61 @@ def test_parse_signed_request_when_correct(self):
         self.assertTrue('algorithm' in result)
 
 
+@mock.patch('requests.request')
```

Comment:
Are these tests specifically meant to verify the changes to POST requests, or are they just general tests to improve library coverage? If they're the latter, they should be submitted as a separate pull request.